rework to be inline with current ADEPT flow

aazaff opened the pull request:
@iross  How do you feel about this a rework fo the instructions?

aazaff committed:
rework to be inline with current ADEPT flow

iross commented on README.md:
```diff
@@ -1,96 +1,58 @@
-- https://github.com/aazaff/usgs_geochron - Mines for geochronology measurements
-- https://github.com/adamancer/speciminer - Finds occurrences of USNM speciments mentioned in the literature
-- https://github.com/throughput-ec/UnacquiredSites/ - Extracts lat/long coordinates from sentences parsed by CoreNLP
+1. **TEXT BASED SEARCHES** The xDD development team has created the **A**utomated **D**ata **E**xtraction **P**la**T**form ([ADEPT](https://xdd.wisc.edu/adept)) to facilitate this process. The ADEPT front-end allows users to browse available documents in the xDD library using full-text search terms (powered by [ElasticSearch](elastic.co)) and other common serach parameters (e.g., publication date ...
```
Link broken.

aazaff replied:
Fxied

iross commented on README.md:
```diff
@@ -1,96 +1,58 @@
 
-## Defining an input 
-In order to deploy an application against the xDD, the _type_ and _set definition_ of input data must be defined. Fundamental questions of each are:
+Requests for development datasets can be made through the [ADEPT browser](https://xdd.wisc.edu/adept) or by contacting the xDD administrative team directly by email (https://contact@geodeepdive.org). Users will then be provided a project and user-specific URL to 200 documents randomly sampled from the identified pool of documents. This link will be provided either by email or through the ADEPT br ...
```
Email shouldn't have "https://"

iross commented on README.md:
```diff
@@ -1,96 +1,58 @@
 
-1. Specifying a list of article DOIs.
-2. Supplying a list of journals to target.
-3. Filtering the corpus by keyword, phrases, publication date ranges, etc.
```
I think it's important to communicate that these kinds of set definitions are possible. Just sending them to ADEPT or the API endpoint doesn't inform potential users what handles exist to define a dataset.

aazaff replied:
Added this back in under obtaining sample data section.

iross reviewed (commented):
See inline comments. I spotted a few broken links and I think we need to communicate what kind of handles a user has to define a set before we send them off to a tool that they need to learn to use.

aazaff committed:
Addressed Ian comments

1. Fixed broken links
2. Added in explicit list of how to define a target set under (Obtaining Sample Data for Development Purposes)
